File: api/app/Models/FaultReportComment.php
<?php
// app/Models/FaultReportComment.php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class FaultReportComment extends Model
{
    protected $fillable = ['fault_report_id', 'user_id', 'comment'];

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class);
    }
}

File: api-net/Models/FaultReportComment.cs
namespace Liftonom.Api.Models;

public class FaultReportComment
{
    public long Id { get; set; }
    public long FaultReportId { get; set; }
    public long? UserId { get; set; }
    public string Comment { get; set; } = "";
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }

    public User? User { get; set; }
}
